src/class-rest.php

```diff
@@ -66,7 +66,7 @@ public function run(): void {
 	 */
 	public function register_parsely_meta(): void {
 		$callback = function( array $object ): array {
-			$post_id = $object['id'];
+			$post_id = $object['ID'] ?? $object['id'];
```

Comment:
I was finding some tests issues as `id` could not be found as a key (but `ID` could.